src/main/java/Duke.java

```diff
@@ -1,10 +1,171 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\____|_|\\_\\___|\n";
+    public static final String HORIZONTAL_LINE = "============================================================";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        greeting();
+        Scanner in = new Scanner(System.in);
+        Task[] list = new Task[100];
+        int counter = 1; //start from 1
+        String userCommand = "";
+        while (!userCommand.contains("bye")) {
+            userCommand = in.nextLine();
+            if (userCommand.contains("list")) {
+                if (counter == 1) {
+                    System.out.println(HORIZONTAL_LINE + "\n");
+                    System.out.println("Your day is clear!");
+                    System.out.println(HORIZONTAL_LINE + "\n");
+                } else {
+                    printListOfTasks(list, counter);
+                }
+            } else if (userCommand.contains("bye")) {
+                bye();
+            } else if (userCommand.contains("unmark")) {
+                unmarkTask(list, userCommand, counter);
+            } else if (userCommand.contains("mark")) {
+                markTask(list, userCommand,counter);
+            } else if (userCommand.contains("todo")) {
+                addTodoTask(list, counter, userCommand);
+                counter++;
+            } else if (userCommand.contains("deadline")) {//Task type D
+                addDeadlineTask(list, counter, userCommand);
+                counter++;
+            } else if (userCommand.contains("event")) {
+                addEventTask(list, counter, userCommand);
+                counter++;
+            }else{
```

Comment:
```suggestion
            } else {
```
Perhaps add spacing around your else for better readability?